editions/tw5.com/tiddlers/$__editions_tw5.com_more-info-view-template.tid

```diff
@@ -0,0 +1,19 @@
+created: 20250326000304980
+list-after: $:/core/ui/ViewTemplate/body
+modified: 20250327020654861
+tags: $:/tags/ViewTemplate
+title: $:/editions/tw5.com/more-info-view-template
```

Comment:
IMO `title: $:/editions/tw5.com/tiddler-related-template`

Reply:
I think of that as simply one of the tabs, although I guess it can refer to other things.  The Wikipedia example that @yaisog shared makes me want to go back to "See Also" for that first tab anyway.